sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/qrcode/SyncBarcodeView.kt

```diff
@@ -86,13 +98,22 @@ constructor(
         ViewModelProvider(findViewTreeViewModelStoreOwner()!!, viewModelFactory)[SquareDecoratedBarcodeViewModel::class.java]
     }
 
+    private var beepManager: BeepManager? = null
+
+    private val cameraSettings = CameraSettings().apply {
```

Comment:
should this also be kill-switch’d ? (separate from `qrCodeScannerHints`)

It seems like an innocent no-brainer of a change, but you never know...